TweetMate/src/main/java/com/ybsystem/tweetmate/activities/TimelineActivity.java

```diff
@@ -55,13 +56,13 @@ private void setTimelineFragment() {
 
         switch (mColumnType) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Complete cases by adding the missing enum constants or add a default case to this switch. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)